src/routes/__tests__/events.test.js

```diff
@@ -0,0 +1,90 @@
+const request = require('supertest');
```

Comment:
Your test is not working for several reasons that are hard to explain. For that, I created a gist that contains a working test for your endpoint. The rest of the test you can do yourself :)
https://gist.github.com/muhammed-shihebi/3b4ab4d00c73f5c78cb76ee681259bff